src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -61,6 +61,11 @@
      */
     private int directionId = -1;
 
+    /**
+     * Route type of the route(s) this alert concerns, set to -1 if no route type.
+     */
+    private int routeType = -1;
```

Comment:
Would it be worth using a constant for this -1 value since it gets used elsewhere in this file? E.g. on line 256 and in AlertsUpdateHandler (line 123).

Reply:
Refactored in d074833db78929d4776ad0ac37724965657cead7